### DISCUSSION TITLE
20231125

### DISCUSSION THREAD

**Discussion by Mailaender** [OpenHV/OpenHV#868] [Announcements]
* Removed screen shake from drop pod explosions.
* Fixed AI not building ore smelters in late game.
* Fixed AI training too few broker pods in late game.
* Fixed submarines being unable to target mine layers.
* Fixed missile sub being unable to target structures under fog.
* Fixed AI building tech buildings too early.
* Fixed force field overlay being too bright.
* Fixed _Crater Defense_ breaches not getting displayed correctly.
* Fixed stone cliffs being traversable.
* Fixed smoke emitter for aircraft.
* Fixed elite pod not targeting air actors.
* Fixed different resource types showing the same tooltip.
* Fixed weapon circle range for bunker.
* Fixed miners not deploying when blocked by friendlies.
* Fixed unloading a vehicle from transport aircraft using the wrong cursor.
* Fixed air transport vehicle and pods conflicting.
* Fixed wrong palette for harbor shadows.
* Fixed _Fire Watch_ mission not ending when all fires are out.
* Fixed map background leaking through UI edges.
* Fixed shore and dirt tiles not displaying hit effects.
* Fixed grass shore tiles being not aligned correctly.
* Fixed faction images for storage buildings.
* Fixed Z offsets for mines.
* Added water trails to submarines.
* Added shadows, centerpieces, corner and damage variants to high walls.
* Added sparks effect for actors targeted midair.
* Added twinkle :sparkles: effects for resources.
* Added explored map as new default for skirmish and multiplayer.
* Added transformation sounds for miners.
* Added upscaled sounds for the radar.
* Added extra silo sprites for miner building.
* Added an audio notification on building capture.
* Added shooting animation for lightning tank.
* Added new map _Deliquescence_ by @SiegeSpud.
* Added lights and details to _Doubles_, _Tournament Island_, _Sandy Shores_, _Tomb Raiders_, _Mount Brosnan_, _Harbor_.
* Added a remastered version of map _Crescendo_.
* Added new crates as decorational props.
* Added reloading sound for minelayers.
* Added a new explosion sound for land and water mines.
* Added big water splash artwork to water mines when they explode.
* Added crane props.
* Added electric pod. :zap: 
* Added high lamp post prop. :bulb: 
* Removed pods ejecting on sell on _Crater Defense_.
* Removed terrain light source as they reveal starting positions.
* Removed WIP artwork mothership.
* Removed manual submerging for submarines.
* Changed lightning effect projectile.
* Changed railgun tank artwork.
* Changed checkbox artwork. :ballot_box_with_check: 
* Changed the layout for in-game encyclopedia. :book: 
* Changed IRC in multiplayer lobbies to automatically connect. :speech_balloon: 
* Changed names to avoid confusion:
    * Hacker pods to Broker :chart_with_upwards_trend: pods as there is already a Hacker tank. :keyboard: 
    * Tech Reconstructor to Nano Reconstructor as it heals everything.
    * Mine to AI Mine as they only trigger on enemies now.
* Changed balancing:
    * Increased storage hit points.
    * Increased harbor and starport energy :electric_plug: requirement.
    * Reduced reload delay, increased speed, removed inaccuracy from Synapol's AA tank.
    * Increased Synapol's AA Turret reload delay.
    * Mothership base is now indestructible.
    * Decreased charge up time for orbital railgun.
    * Bomber pod is now Synapol exclusive.
    * Reduced hit points of Ramp Buggy and Gatling Bike.
    * Increased Ramp Buggy and Gatling Bike weapon's effectiveness.
    * Increased Gatling Bike's moving and turn speed.

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenHV/OpenHV/releases/tag/20231125'>20231125</a>.</em>

**Comment by Maguster7** [comment]
No launching game
